src/Filter/NumberFilter.php

```diff
@@ -19,32 +19,28 @@
 
 class NumberFilter extends Filter
 {
-    /**
-     * @param string $alias
-     * @param string $field
-     * @param string $data
-     */
+    private const CHOICES = [
+        NumberOperatorType::TYPE_EQUAL => 'equals',
+        NumberOperatorType::TYPE_GREATER_EQUAL => 'gte',
+        NumberOperatorType::TYPE_GREATER_THAN => 'gt',
+        NumberOperatorType::TYPE_LESS_EQUAL => 'lte',
+        NumberOperatorType::TYPE_LESS_THAN => 'lt',
+    ];
+
     public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $data)
     {
         if (!$data || !\is_array($data) || !\array_key_exists('value', $data) || !is_numeric($data['value'])) {
             return;
         }
 
-        $type = $data['type'] ?? false;
-
-        $operator = $this->getOperator($type);
+        $type = $data['type'] ?? NumberOperatorType::TYPE_EQUAL;
 
-        if (!$operator) {
-            $operator = 'equals';
-        }
+        $operator = $this->getOperator((int) $type);
 
         $queryBuilder->field($field)->$operator((float) $data['value']);
         $this->active = true;
     }
 
-    /**
-     * @return array
```

Comment:
why remove this?

Reply:
[Same thing](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Filter/FilterInterface.php#L43)